### ISSUE TITLE
Launching Windows Applications more easily

### ISSUE THREAD

**Issue by psharma04** [NathanCastle/BootShellCredentialProvider#6]
Currently, attempting to launch a Windows .exe application will request to install Wine. Is there any way to make Windows executables launchable from the Linux DEs?

**Comment by kragebein** [comment]
No. You have to run them from /mnt/c in the windows hiearchy. That way the Application will run as normal.
